template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Data/SeedClientApplication.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Company.WebApplication1.Identity.Data;
+using JetBrains.Annotations;
+
+namespace Company.WebApplication1.Identity.Data.Migrations
+{
+    public static class SeedClientApplication
+    { 
+        public static void Seed(MigrationBuilder migrationBuilder)
+        {
+            var applicationColumns = new string[] { "Id", "ClientId", "Name" };
+            var appId = Guid.NewGuid().ToString();
+            var clientId = "53bc9b9d-9d6a-45d4-8429-2a2761773502";
+
+            var applicationValues = new string[1, 3]
+            {
+                {appId, clientId, "Company.WebApplication1" }
+            };
+
+            migrationBuilder.Insert("AspNetApplications", null, applicationColumns, applicationValues);
```

Comment:
I don't like all the local variables. When EF starts generating these, they'll look like this:
```C#
migrationBuilder.Insert(
    table: "AspNetApplications",
    columns: new[] { "Id", "ClientId", "Name" },
    values: new object[,]
    {
        { "1e3ff518-502e-42e7-ba6a-d703c92eb1da", "53bc9b9d-9d6a-45d4-8429-2a2761773502", "Company.WebApplication1" }
    });    
```